/*
 * ConfigInc.h
 *
 *  Created on: 2016-8-22
 *      Author: Ralf
 */

#ifndef CONFIGINC_H_
#define CONFIGINC_H_

#include "Demo.pb.h"
#include "CommonGiftConfig.pb.h"
#include "UserConfig.pb.h"
#include "ConfigItem.pb.h"
#include "Hero.pb.h"
#include "Combine.pb.h"
#include "Stars.pb.h"
#include "Archive.pb.h"
#include "DragonBall.pb.h"
#include "BattleDemoConfig.pb.h"
#include "Stages.pb.h"
#include "AddWeight.pb.h"
#include "Grade.pb.h"
#include "NpcConfig.pb.h"
#include "Random.pb.h"
#include "FriendConfig.pb.h"
#include "Onhook.pb.h"
#include "MartialClub.pb.h"
#include "RobotConfig.pb.h"
#include "ActivityConfig.pb.h"
#include "Cards.pb.h"


#endif /* CONFIGINC_H_ */
